extensions/blocks/rating-star/rating-meta.php

```diff
@@ -14,7 +14,7 @@
 	 * @return string
 	 */
 	function jetpack_rating_meta_get_symbol_low_fidelity() {
-		return '⭐';
```

Comment:
For some reason, AMP doesn't like introducing a `span` here and won't render the symbols (try by adding `/amp` at the end of the testing post).

Reply:
Actually, the reason this isn't displayed is because this inline CSS for AMP 0.6.2 (see `rating-star.php`):

```php
/**
 * The following filter is added only to support the old 0.6.2 version of the AMP plugin.
 * This entire section can be removed once we're on version a newer version.
 * Confirmed that version 1.4.1 (or presumably newer) does not need this filter.
 */
function jetpack_rating_star_amp_add_inline_css() {
	echo '.wp-block-jetpack-rating-star span { display: none; }';
}
add_action( 'amp_post_template_css', 'jetpack_rating_star_amp_add_inline_css', 11 );
```

Reply:
Just checked and it doesn't look like we've updated to the latest AMP version, so we still need this code.

Reply:
Noted, will leave as is for now.

Reply:
Sorry I wasn't clear in my request! I meant that we can't add the `span` here without also modifying the `jetpack_rating_star_amp_add_inline_css`, or perhaps remove the span altogether. I hadn't had a chance to look deeply into this code, so I don't know what would be best. I saw that you're about to take an extended leave, so I'm happy to help here if you don't have the time for this.

Reply:
@nosolosw scottsweb seems to be offline still and this PR is left hanging. Would it be possible for you to help with a fix for the AMP problem so this PR could be merged?